Resources/Prototypes/_Moffstation/PersonalItems/Wearables/Pixel8/Marvin_Dugmore/marvinheadphones.yml

```
@@ -0,0 +1,41 @@
+- type: entity
+  parent: ClothingMultipleHeadphones
+  id: PersonalItemMarvinHeadphones
+  name: unusual noise cancelling headphones
+  description: A pair of headphones with a clip-on microphone made to help those with sensitive ears. Something seems off about this pair.
+  suffix: PersonalItem
+  components:
+  - type: Item
+    sprite: _Moffstation/PersonalItems/Wearables/Pixel8/Marvin_Dugmore/marvinheadphones.rsi
+  - type: Sprite
+    sprite: _Moffstation/PersonalItems/Wearables/Pixel8/Marvin_Dugmore/marvinheadphones.rsi
+  - type: Clothing
+    quickEquip: True
+    sprite: _Moffstation/PersonalItems/Wearables/Pixel8/Marvin_Dugmore/marvinheadphones.rsi
+
+  - type: PointLight
+    enabled: false
+    netsync: false
+    radius: 1.1 # mimics a flippo without any of the "set you on fire" capability. More of a flavor thing, since the PDA light is brigher
+    color: orange
+  - type: ItemTogglePointLight
+  - type: ItemToggle
+    predictable: false
```

```suggestion
  - type: ItemToggle
```
I think this should be fine.

---

Alrighty, if that's all that needs fixing up, then it should be working as intended now 👍